custom-words.txt

```diff
@@ -2061,3 +2061,21 @@ tpgs
 ProviderHub
 checkin
 regionality
+Connectionand
```

Comment:
Some of these are actual typos that should be fixed or ignored for a specific version rather than globally ignored

Reply:
Thanks, I just made some updates to both files.  Let me know if you notice any other issues here.

Reply:
Nice! I'm trying to update the instructions over in https://github.com/Azure/azure-rest-api-specs/pull/12624

Reply:
Thanks for sharing!  Based on this, I'm wondering if I should instead add an override for Text Analytics in cSpell.json, since most of what I added are acronyms and not real words.

Reply:
I'm not an MS employee, I just setup the linting years ago and have been poking around to see the state it's in recently, so go with whatever the MS folks agree with😄

Reply:
Ok, I think I will leave it as-is for now.  This PR is meant for solving some CI issues and its contents will undergo a final review before everything is eventually merged to master.  :)